web/packages/teleterm/src/services/pty/types.ts

```diff
@@ -41,6 +41,10 @@ export type PtyServiceClient = {
 export type ShellCommand = PtyCommandBase & {
   kind: 'pty.shell';
   cwd?: string;
+  // env is a record of additional env variables that need to be set for the shell terminal and it
+  // will be merged with process env.
+  env?: Record<string, string>;
+  helpMsg?: string;
```

Comment:
Let's add a comment explaining what `helpMsg` is, with the focus on the fact that it's not either written nor read from the underlying PTY.